### PR TITLE
Feature/mdr 358

### PR DESCRIPTION
**Pull request by Dragos0000**

### CODE REVIEW

File: rdf_differ/entrypoints/api/handlers.py [hunk 1]
```diff
@@ -153,25 +153,33 @@ def delete_diff(dataset_id: str) -> tuple:
...
+    if application_profile not in config.RDF_DIFFER_APPLICATION_PROFILES_LIST:
+        exception_text = f"The chosen application profile does not exist. Existing application profiles:" \
+                         f" {','.join(config.RDF_DIFFER_APPLICATION_PROFILES_LIST)}"
+        logger.exception(exception_text)
+        raise UnprocessableEntity(exception_text)
+
     try:
         with tempfile.TemporaryDirectory() as temp_dir:
-            template_location = config.RDF_DIFFER_REPORT_TEMPLATE_LOCATION
+            template_location = config.RDF_DIFFER_REPORT_TEMPLATE_LOCATION + "/" + application_profile
```
Comment (cosletmihai) [inline rdf_differ/entrypoints/api/handlers.py:176]: I think we should delegate this logic to the `config` module. 
```python
def get_aplication_profile_location(application_profile):
    return f'{RDF_DIFFER_REPORT_TEMPLATE_LOCATION}/{application_profile}'
```
and call it like:
```python
template_location = config.get_aplication_profile_location(application_profile)

File: rdf_differ/entrypoints/api/handlers.py [hunk 2]
```diff
@@ -153,25 +153,33 @@ def delete_diff(dataset_id: str) -> tuple:
...
             logger.debug(f'template location {template_location}')
             copytree(template_location, temp_dir, dirs_exist_ok=True)
+            logger.debug(f'template location {temp_dir}')
```
Comment (cosletmihai) [inline rdf_differ/entrypoints/api/handlers.py:179]: remove this log

**Review by cosletmihai — APPROVED** [review]

File: rdf_differ/entrypoints/api/handlers.py (hunk 2 above)
Comment (Dragos0000) [reply rdf_differ/entrypoints/api/handlers.py:179]: done

File: rdf_differ/entrypoints/api/handlers.py (hunk 1 above)
Comment (Dragos0000) [reply rdf_differ/entrypoints/api/handlers.py:176]: done

**Pull request merged**